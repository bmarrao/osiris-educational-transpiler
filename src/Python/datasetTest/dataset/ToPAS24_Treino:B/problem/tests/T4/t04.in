1 1
24 7
2 K
