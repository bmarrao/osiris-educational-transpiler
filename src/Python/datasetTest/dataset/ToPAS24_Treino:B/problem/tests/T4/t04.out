7 7 7 7 7 7 7 7 7 7 7 7 7 7 7 7 7 7 7 7 7 7 7 7
K K
